test/studies/shootout/chameneos-redux/elliot/chameneosredux-ejr.chpl

```diff
@@ -0,0 +1,291 @@
+/* The Computer Language Benchmarks Game
+   http://benchmarksgame.alioth.debian.org/
+
+   contributed by Hannah Hemmaplardh, Lydia Duncan, and Brad Chamberlain
+   derived in part from the GNU C version by Dmitry Vyukov
+*/
+
+config const n = 600,              // number of meetings (must be >= 0)
+             popSize1 = 3,         // size of population 1 (must be > 1)
+             popSize2 = 10;        // size of population 2 (must be > 1)
+
+enum Color {blue=0, red, yellow};  // the chameneos colors
+use Color;                         // permit unqualified references to them
+
+
+//
+// Print the color equations and simulate the two population sizes.
+//
+proc main() {
+  printColorEquations();
+
+  const group1 = new Population(popSize1);
+  const group2 = new Population(popSize2);
+
+  cobegin {
+    group1.holdMeetings(n);
+    group2.holdMeetings(n);
+  }
+
+  group1.printColors();
```

Comment:
Elliot, do you understand how printing the colors after the meetings is generating the same result as our previous version which prints the colors before holding the meetings?  I can't figure out what I must be missing...